src/SimpleApplication.mjs

```diff
@@ -0,0 +1,63 @@
+/**
+ * A simplified implementation of `ApplicationV2` that allows for easy rendering of
+ * arbitrary HTML content.
+ * @class
+ * @type {new foundry.applications.api.ApplicationV2}
+ */
+export let SimpleApplication;
+
+try {
+  SimpleApplication = class extends foundry.applications.api.ApplicationV2 {
```

Comment:
suggestion: rename to something more specific

The name SimpleApplication can mean a lot of things, if its done to help with `easy rendering of arbitrary HTML content` I would name it HTMLContentApplication or something along those lines.

Reply:
That's a really *ugly* name though, and I don't think the name needs to tell you anything about the implementation. It's basically an alternative to `Dialog` now that `DialogV2` is heavily restricted. And while it *could* mean a lot of things, so could "Dialog" or "Application". I had considered `SimpleForgeApplication` or just `ForgeApplication` but then remembered that we're using esmodules and don't need to worry about name collisions.

Reply:
It tells you nothing, and while I get that the name may be ugly it tells you something more. Names should not be pretty, names should be informative, even without JSDoc.